lib/github_changelog_generator/generator/generator_fetcher.rb

```diff
@@ -12,13 +12,13 @@ def fetch_events_for_issues_and_pr
     end
 
     # Async fetching of all tags dates
```

Comment:
Make this more functional, so we can track what's going on, instead of modifying the instance variable.